### ISSUE TITLE
Radial nerve distribution to findings

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10792]
**Preferred term label:**
Radial nerve distribution to findings

**Synonyms**
Paresthesias on the dorsoradial aspect of the hand and wrist, including the dorsal thumb, index finger and middle finger. Weakness in elbow extension, wrist extension, finger extension, thumb extension and/or supination of the forearm.

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
There are entries that are similar for other nerves:
Median: https://hpo.jax.org/browse/term/HP:0012185
Ulnar: https://hpo.jax.org/browse/term/HP:0012186

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Posterior interosseous nerve syndrome (PINS)
Superficial radial nerve (SRN) compression

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001176
- label: Radial nerve distribution
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to the localization of findings corresponding to the distribution of the radial nevrve, such as (i)paresthesias on the dorsoradial aspect of the hand and wrist, including the dorsal thumb, index finger and middle finger and (ii) Weakness in elbow extension, wrist extension, finger extension, thumb extension and/or supination of the forearm.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10792
- synonyms:
